app/Models/Competition.php

```diff
@@ -12,14 +11,14 @@ class Competition extends Model
      *
      * @var array
      */
-    protected $fillable = ['campaign_id', 'campaign_run_id', 'start_date', 'end_date'];
+    protected $fillable = ['contest_id', 'competition_start_date', 'competition_end_date'];
```

Comment:
Would the system autofill it at all? Given it's creating a competition for a specified contest? Or do we need to set the contest id each time a competition is created, which is fine too, just more work...